nextjs-blog/components/decisioncard.js

```diff
@@ -1,19 +1,35 @@
 import React from "react";
 import Head from "next/head";
-import Textbox from "../components/textbox";
 import styles from "../styles/decisioncard.module.scss";
 import CardImage from "../components/cardimage";
 import TypingTextBox from "../components/typingtextbox";
+import dbMocks from "../__mocks__/dbMocks.json";
 
-export default function DecisionCard() {
+export default function DecisionCard({
+  currentevent,
+  setCurrentevent,
+  setHealthone,
+  setHealthtwo,
+  setHealththree,
+  setHealthfour,
+}) {
+  console.log(dbMocks);
+  console.log(currentevent);
   return (
     <>
       <Head>
         <title>Decision Card Container</title>
       </Head>
       <div id="decisioncardcontainer" className={styles.CardAlpha}>
-        <TypingTextBox text="I Am Here..... And I can type myself!" />
-        <CardImage imageUrl="/pixelholder.jpg" />
+        <TypingTextBox text={dbMocks[currentevent].text} />
+        <CardImage
```

Comment:
Just for knowledge, it is possible to declare props and pass them into a child component in an object